lib/rock/webapp/tasks/api.rb

```diff
@@ -8,6 +9,12 @@ class API < Grape::API
                 version 'v1', using: :header, vendor: :rock
                 format :json
 
+                if ENV["REST_API_LOG_CONSOLE"]
+                    puts "logging"
```

Comment:
well, forgot to explain: the grape_logging package installes ad grape middleware to log the request mechanism of grape itself (logs POST/GET requests), there is no method to log custom messages